shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -84,13 +76,23 @@ FLUTTER_DARWIN_EXPORT
 @property(nonatomic) FlutterMouseTrackingMode mouseTrackingMode;
 
 /**
- * Initializes a controller that will run the given project.
+ * Initializes this `FlutterViewController` for the implicit view of a new
+ * `FlutterEngine`.
  *
- * In this initializer, this controller creates an engine, and is attached to
- * that engine as the default controller. In this way, this controller can not
- * be set to other engines. This initializer is suitable for the first Flutter
- * view controller of the app. To use the controller with an existing engine,
- * use initWithEngine:nibName:bundle: instead.
+ * This initializer is a shorthand for creating an engine explicitly and
+ * initializing with `initWithEngine:nibName:bundle:`.
```

Comment:
I worry this is a little too low-level and might confuse readers. What do you think of simplifying? Perhaps:

```suggestion
 * Initializes a controller that will run the given project.
 *
 *  In this initializer, this controller creates an engine, and is attached to
 * that engine as the implicit view.
```